inference_experimental/tests/integration_tests/models/test_perception_encoder.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from inference_exp.models.perception_encoder.perception_encoder import (
+    PerceptionEncoder,
+)
+
+
+def test_perception_encoder_text_embedding():
+
+    #TODO: this is a temporary path, should be replaced with a proper path / registry integration
+    model = PerceptionEncoder.from_pretrained(
+        "/tmp/cache/perception_encoder/PE-Core-B16-224" 
```

Comment:
just have this locally t the moment so I could do sanity check (to download can use PE in workflow block with existing inference) until added to registry